rust/BLAKE3/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "_ch_rust_blake3"
+version = "0.1.0"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+blake3 = "1.2.*"
+libc = "0.2.*"
+
+[lib]
+crate-type = ["staticlib"]
```

Comment:
With `cdylib` enabled, corrosion adds some standard libraries even if NO_STD flag is used. This can cause errors during Rust library linking, so I decided to leave only `staticlib` for now.